driver/vhci/vhci_write.c

```diff
@@ -232,6 +232,12 @@ store_urb_data(PURB urb, struct usbip_header *hdr)
 	case URB_FUNCTION_ISOCH_TRANSFER:
 		status = store_urb_iso(urb, hdr);
 		break;
+	case URB_FUNCTION_GET_STATUS_FROM_DEVICE:
+	case URB_FUNCTION_GET_STATUS_FROM_INTERFACE:
+	case URB_FUNCTION_GET_STATUS_FROM_ENDPOINT:
+	case URB_FUNCTION_GET_STATUS_FROM_OTHER:
+		status = store_urb_control(urb, hdr);
```

Comment:
It's trivial but how about merging with other control cases?
All other codes are great!

Reply:
Yes sure

Reply:
I hope that is what you meant, please double check.
Thanks.